gating/generate_release_notes/run

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash -xe
+
+# This script is called by Jenkins to generate release notes.
+
+# The resulting notes must be written to $WORKSPACE/artifacts/release_notes
+
+# It runs in docker as it requires pandoc which is an apt package, and
+# apt packages can't be installed on shared Jenkins slaves.
+
+tag="${BUILD_TAG:-rpco_reno_$(date +%d%m%Y_%H%M)}"
```

Comment:
The use of `tag` and `tag_filtered` threw me slightly because there is also `PREVIOUS_TAG` and `NEW_TAG`, is there a better name that could be used here?

Reply:
Prepended docker_ to docker tag related variables. I can see how git tags and docker tags could be confused.